### DISCUSSION TITLE
Is there a reason that the latest releases fs2-aws library are not being pushed to repositories

### DISCUSSION THREAD

**Discussion by scottexemine** [laserdisc-io/fs2-aws#957] [Q&A] [answered]
When I try to use the latest release 5.1.0 of this library I am not able to find it in Maven Central ( https://mvnrepository.com/artifact/io.laserdisc/fs2-aws ) the latest version that has been pushed is 3.1.1 on May 5, 2021. 
Is there some reason that the later versions are not being pushed or should I get this version from a different repository?

**Comment by semenodm** [comment, marked as answer]
Hello. We did a significant reorg of this project, now every streaming resource is represented as a separate module and package. If you want to use Kinesis components, you should use `fs2-aws-kinesis`, if it is SQS, use `fs2-aws-sqs`, and so on. Let me know if you have trouble importing these modules
